diff --git a/scripts/exports/ffmpeg_h264.sh b/scripts/exports/ffmpeg_h264.sh
index bfc08e3..76f69b2 100755
--- a/scripts/exports/ffmpeg_h264.sh
+++ b/scripts/exports/ffmpeg_h264.sh
@@ -53,24 +53,24 @@ execute()
 	case "$profile" in 
 		"0" ) 	ffmpeg -threads $threads -f dv -i pipe: \
 			-vcodec $x264 -g 300 $hq $interlace -aspect $aspect -qscale 2 -ab 192$audio_kilo \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"1" ) 	ffmpeg -threads $threads -f dv -i pipe: \
 			-vcodec $x264 -g 300 $hq $progressive -s $full_res -aspect $aspect -qscale 2 -ab 192$audio_kilo \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"2" ) 	ffmpeg -threads $threads -f dv -i pipe: \
 			-vcodec $x264 -g 300 $hq $progressive -s $med_res -aspect $aspect -qscale 4 -ab 128$audio_kilo -ar 44100 \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"3" ) 	ffmpeg -threads $threads -f dv -i pipe: \
 			-vcodec $x264 -g 300 $hq $progressive -s $med_res -aspect $aspect -b 500$kilo -ab 64$audio_kilo -ar 32000 \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"4" ) 	ffmpeg -threads $threads -f dv -i pipe: \
 			-vcodec $x264 $hq -r 12 -g 120 $progressive -s qcif -aspect $aspect -b 90$kilo -minrate 0 -maxrate 100$kilo -bufsize 40$bufsize \
 			-ac 1 -ab 28$audio_kilo -ar 16000 \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 	esac
 }
diff --git a/scripts/exports/ffmpeg_h264_dual.sh b/scripts/exports/ffmpeg_h264_dual.sh
index 021576a..fec4981 100755
--- a/scripts/exports/ffmpeg_h264_dual.sh
+++ b/scripts/exports/ffmpeg_h264_dual.sh
@@ -58,24 +58,24 @@ execute()
 	case "$profile" in 
 		"0" ) ffmpeg -threads $threads -f dv -i pipe: -pass $pass -passlogfile "$file" \
 			-vcodec $x264 -g 300 $hq $interlace -aspect $aspect -b 2048$kilo -ab 192$audio_kilo \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"1" ) ffmpeg -threads $threads -f dv -i pipe: -pass $pass -passlogfile "$file" \
 			-vcodec $x264 -g 300 $hq $progressive -s $full_res -aspect $aspect -b 2048$kilo -ab 192$audio_kilo \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"2" ) ffmpeg -threads $threads -f dv -i pipe: -pass $pass -passlogfile "$file" \
 			-vcodec $x264 -g 300 $hq $progressive -s $med_res -aspect $aspect -b 1024$kilo -ab 128$audio_kilo -ar 44100 \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"3" ) ffmpeg -threads $threads -f dv -i pipe: -pass $pass -passlogfile "$file" \
 			-vcodec $x264 -g 300 $hq $progressive -s $med_res -aspect $aspect -b 500$kilo -ab 64$audio_kilo -ar 32000 \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"4" ) ffmpeg -threads $threads -f dv -i pipe: -pass $pass -passlogfile "$file" \
 			-vcodec $x264 $hq $progressive -r 12 -g 120 -s qcif -aspect $aspect -b 90$kilo -minrate 0 -maxrate 100$kilo -bufsize 40$bufsize \
 			-ac 1 -ab 24$audio_kilo -ar 16000 \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 	esac
 	[ $pass -eq "2" ] && rm -f "$file-0.log"
diff --git a/scripts/exports/ffmpeg_h264_ps3.sh b/scripts/exports/ffmpeg_h264_ps3.sh
index f8c357b..b9f1609 100755
--- a/scripts/exports/ffmpeg_h264_ps3.sh
+++ b/scripts/exports/ffmpeg_h264_ps3.sh
@@ -36,25 +36,12 @@ execute()
 	# generate filename if missing
 	[ "x$file" = "x" ] && file="kino_export_"`date +%Y-%m-%d_%H.%M.%S`
 	
-	# Create metadata options
-	title=`awk '/title="/ {split($0, x, "\""); z = 0; for (y in x) if (x[y] ~ /title=/) z = y + 1; print x[z]; exit}' "$project"`
-	author=`awk '/author="/ {split($0, x, "\""); z = 0; for (y in x) if (x[y] ~ /author=/) z = y + 1; print x[z]; exit}' "$project"`
-	comment=`awk '/abstract="/ {split($0, x, "\""); z = 0; for (y in x) if (x[y] ~ /abstract=/) z = y + 1; print x[z]; exit}' "$project"`
-	copyright=`awk '/copyright="/ {split($0, x, "\""); z = 0; for (y in x) if (x[y] ~ /copyright=/) z = y + 1; print x[z]; exit}' "$project"`
-	
 	# Run the command
 	 case "$pass" in
 		"1" ) ffmpeg -threads 0 -f dv -i pipe: -pass 1 -passlogfile "$file" \
 			-vcodec $x264 -vpre fastfirstpass -level 41 -g 300 $interlace -aspect $aspect -b 2048$kilo -bt 4096$kilo \
 			-an -f rawvideo -y /dev/null
 			;;
-
-		"2" ) ffmpeg -threads 0 -f dv -i pipe: -pass 2 -passlogfile "$file" \
-			-vcodec $x264 -vpre hq -level 41 -g 300  $interlace -aspect $aspect -b 2048$kilo -bt 4096$kilo \
-			-ab 192$audio_kilo -ar 44100 -title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
-			-y "$file".mp4
-			rm -f "$file-0.log"
-			;;
 	esac
 }
 
diff --git a/scripts/exports/ffmpeg_mp4.sh b/scripts/exports/ffmpeg_mp4.sh
index 368fe1d..568caa4 100755
--- a/scripts/exports/ffmpeg_mp4.sh
+++ b/scripts/exports/ffmpeg_mp4.sh
@@ -45,26 +45,26 @@ execute()
 	copyright=`awk '/copyright="/ {split($0, x, "\""); z = 0; for (y in x) if (x[y] ~ /copyright=/) z = y + 1; print x[z]; exit}' "$project"`
 	
 	# Run the command
-	case "$profile" in 
+	case "$profile" in
 		"0" ) 	ffmpeg -threads $threads -f dv -i pipe: \
 			-g 300 $hq $interlace -aspect $aspect -qscale 2 -ab 192$audio_kilo \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"1" ) 	ffmpeg -threads $threads -f dv -i pipe: \
 			-g 300 $hq $progressive -s $full_res -aspect $aspect -qscale 2 $qpel -ab 192$audio_kilo \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"2" ) 	ffmpeg -threads $threads -f dv -i pipe: \
 			-g 300 $hq $progressive -s $med_res -aspect $aspect -qscale 4 -ab 128$audio_kilo -ar 44100 \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"3" ) 	ffmpeg -threads $threads -f dv -i pipe: \
 			-g 300 $hq $progressive -s $med_res -aspect $aspect -b 500$kilo -g 60 -ab 64$audio_kilo -ar 32000 \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"4" ) 	ffmpeg -threads $threads -f dv -i pipe: \
 			$hq -r 12 -g 120 $progressive -s $low_res -aspect $aspect -b 96$kilo -ac 1 -ab 32$audio_kilo -ar 22050 \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 	esac
 }
diff --git a/scripts/exports/ffmpeg_mp4_dual.sh b/scripts/exports/ffmpeg_mp4_dual.sh
index f393fa5..b9cddc9 100755
--- a/scripts/exports/ffmpeg_mp4_dual.sh
+++ b/scripts/exports/ffmpeg_mp4_dual.sh
@@ -51,23 +51,23 @@ execute()
 	case "$profile" in 
 		"0" ) 	ffmpeg -threads $threads -f dv -i pipe: -pass $pass -passlogfile "$file" \
 			-g 300 $hq $interlace -aspect $aspect -b 2048$kilo -ab 192$audio_kilo \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"1" ) 	ffmpeg -threads $threads -f dv -i pipe: -pass $pass -passlogfile "$file" \
 			-g 300 $hq $progressive -s $full_res -aspect $aspect -b 2048$kilo -ab 192$audio_kilo \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"2" ) 	ffmpeg -threads $threads -f dv -i pipe: -pass $pass -passlogfile "$file" \
 			-g 300 $hq $progressive -s $med_res -aspect $aspect -b 1024$kilo -ab 128$audio_kilo -ar 44100 \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"3" ) 	ffmpeg -threads $threads -f dv -i pipe: -pass $pass -passlogfile "$file" \
 			-g 300 $hq $progressive -s $med_res -aspect $aspect -b 500$kilo -g 60 -ab 64$audio_kilo -ar 32000 \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 		"4" ) 	ffmpeg -threads $threads -f dv -i pipe: -pass $pass -passlogfile "$file" \
 			$hq -r 12 -g 120 $progressive -s $low_res -aspect $aspect -b 96$kilo -ac 1 -ab 32$audio_kilo -ar 22050 \
-			-title "$title" -author "$author" -comment "$comment" -copyright "$copyright" \
+			${opt_title}"${title}" ${opt_author}"${author}" ${opt_comment}"${comment}" ${opt_copyright}"${copyright}" \
 			-y "$file".mp4 ;;
 	esac
 	[ $pass -eq "2" ] && rm -f "$file-0.log"
diff --git a/scripts/exports/ffmpeg_utils.sh b/scripts/exports/ffmpeg_utils.sh
index 9c6c308..6e149e6 100644
--- a/scripts/exports/ffmpeg_utils.sh
+++ b/scripts/exports/ffmpeg_utils.sh
@@ -31,6 +31,12 @@ if [ "$1" != "--usage" ] && [ "x$1" != "x" ]; then
 		med_res=`[ "$normalisation" = "pal" ] && echo "384x288" || echo "320x240"`
 		low_res=`[ "$normalisation" = "pal" ] && echo "192x144" || echo "160x120"`
 	fi
+
+	test_metadata=`echo $ffmpeg_help | grep '\-metadata' | wc -l`
+	opt_title=`[ "$test_metadata" -gt 0 ] && echo "-metadata title=" || "-title "`
+	opt_author=`[ "$test_metadata" -gt 0 ] && echo "-metadata author=" || "-author "`
+	opt_comment=`[ "$test_metadata" -gt 0 ] && echo "-metadata comment=" || "-comment "`
+	opt_copyright=`[ "$test_metadata" -gt 0 ] && echo "-metadata copyright=" || "-copyright "`
 fi
 
 # Generate FFMPEG hiqh quality settings based upon version
